### ISSUE TITLE
The progress status

### ISSUE THREAD

**Issue by huyduong7101** [johndpope/VASA-1-hack#15]
Hi @johndpope , have you tried to train it yet? If you did, could you share a little about losses and metrics during training?
I am building a model that mixes VASA-1 and AniPortrait. I adopted only the diffusion loss (MSELoss). During training, the loss is fluctuated between 0.1 and 0.3, and the final results (video) is irrelevant between frames.

**Comment by johndpope** [comment]
I create a draft PR which is based off work from megapaportraits. Emportraits code will drop next month - maybe not training code. I’m not sure exactly what the rendering engine - is it using stable diffusion pipeline? Ddpm? I have the diffusion transformer in this code - and 2 training stages. I’m very interested to know your thoughts on that branch- my main is based on key points which MS didn’t use.

**Comment by johndpope** [comment]
i merge this with fresh updates from sonnet. there's a lot of work to do still.
